.github/workflows/deploy.yml

```diff
@@ -0,0 +1,51 @@
+name: build
+
+on: 
+  workflow_call:
```

Comment:
What does this do?

Reply:
Only runs when something explictly calls it, see the cd.yaml file

Reply:
If you mean the environment input, right now its specifying which dispatch event to trigger on the child repo